kge/model/kge_model.py

```diff
@@ -431,6 +445,33 @@ def create_from(
         model.eval()
         return model
 
+    def init_pretrained(self, packaged_model: Dict):
+        entity_package = {
+            "model": dict(
+                [
+                    (k.split("_entity_embedder.")[1], v)
+                    for k, v in packaged_model["model"][0].items()
+                    if k.startswith("_entity_embedder")
```

Comment:
This won't work (see good_keys above). 

Option 1: check whether there is exactly one key that ends with ._entity_embedder.embeddings.weight and use that one.
Option 2: just load the model, the get its entity embedder (cleaner, I think).

Reply:
implemented option 2